streamlitDemo/README.md

```diff
@@ -19,9 +19,9 @@ Now, the web application will be running and accessible through the following UR
 
 ## API Setup 
 Here is the list of APIs that must be running to access the complete functionality of the web application. Note that each of these APIs is contained within their respective notebooks, and you need to have them running:
-- Speech-to-Text API
-- Text-to-Speech API
-- Question-Answering API
+- [Speech-to-Text API](https://github.com/nsmq-ai/nsmqai/blob/main/speech-to-text/STT_Inference_API_Server_Colab_Notebook.ipynb)
```

Comment:
I think you should be also able to relative links instead of the blob url https://github.blog/2013-01-31-relative-links-in-markup-files/